### ISSUE TITLE
AutomationProperty.Name doesn't work for ComboBox when IsEditable=True

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#1206]
Speak with Accessibility team and WPF about why AutomationProperty.Name doesn't work for a ComboBox with a text box inside.  See more about ComboBox when IsEditable= true [here](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.combobox.iseditable?view=net-5.0).

**Comment by zewditu** [comment]
Temporary fixes are done  by https://github.com/dotnet/machinelearning-tools/pull/932 and  https://github.com/dotnet/machinelearning-tools/pull/878/files

**Comment by beccamc** [comment]
Closing this as I've removed the workaround

**Comment by beccamc** [comment]
For others who may come across this WPF issue. 

**Root cause:**  When a combobox is marked as editable, an underlying edit control is added to the UIA tree as a child of the combobox. This edit control is the one that receives focus, which means narrator reads the properties of the edit control rather than the ones from the combobox. 

**Work around:** 
For whatever reason AutomationProperty.Name doesn't work, but AutomationProperty.LabeledBy does work. We added a collapsed label containing the automation property name. 

```
<Label Name="SubscriptionsLabel" Content="{x:Static strings:AzureAttachResources.SubscriptionLabel}" Visibility="Collapsed"/>
<ComboBox
  Name="SubscriptionComboBox"
  AutomationProperties.HelpText="{x:Static strings:AzureAttachResources.SubscriptionsTooltip}"
   AutomationProperties.LabeledBy="{Binding ElementName=SubscriptionsLabel}"
   IsEditable="True"
   IsEnabled="{Binding SubscriptionComboBoxState.IsEnabled}"/>
```
